.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish package to the Maven Central Repository
+on:
```

Comment:
we might want to create a trigger temporary to be able to run the job and test the publication without the need to create a release